docs/configuration/configuration.md

```diff
@@ -451,6 +451,26 @@ metric_relabel_configs:
 # native histogram. If this is exceeded, the entire scrape will be treated as
 # failed. 0 means no limit.
 [ native_histogram_bucket_limit: <int> | default = 0 ]
+
+# Lower limit for the growth factor of one bucket to the next in each native histogram. 
+# The resolution of a histogram with a lower growth factor will be reduced until it is within the limit.
+# To set an upper limit for the schema (equivalent to "scale" in OTel's exponential histograms), use the following factor limits:
+# | growth factor | resulting schema AKA scale |
```

Comment:
I have just realized that this is not rendered as markdown, but as pure text. So this table should probably get a "blank" line (just an `#`) to separate it from the text more clearly, and should be formatted in true ASCII-art style so that the column widths match.